### ISSUE TITLE
bug in test

### ISSUE THREAD

**Issue by ValWood** [pombase/canto#1128]
I'm adding a phenotype to a multi gene genotype.
It seems to go OK, to here:

![grab](https://cloud.githubusercontent.com/assets/7359272/14982646/bd9576e0-112f-11e6-8417-82cc473e5d6e.jpg)

and then I gat the 'disallowed symbol" when I go over 'proceed'

**Comment by ValWood** [comment]
I tried again and got past this part, but then I could not proceed from this stage:
![cant proceed](https://cloud.githubusercontent.com/assets/7359272/14982736/5ba4f996-1130-11e6-978e-c13c51ae7c55.jpg)

This ones odd because the annotation has appeared, but there is still a text box (which I don't want to add anything to?)

**Comment by kimrutherford** [comment]
I can't reproduce this.  Has it happened more than once?  Could you demo it to me sometime?

**Comment by mah11** [comment]
Test won't save any new phenotype annotations for me.

I'm getting something similar to the above for the step-by-step phenotype interface. It won't let me proceed normally beyond the "finish annotation" page, and if I go to the paper summary via the breadcrumb link, everything is normal except that the new annotation hasn't been saved.

Yesterday I thought I also saw a thing where it didn't save if I duplicated an existing phenotype annotation, but that bit was OK just moments ago.

As far as I can tell, only phenotype annotations are affected. I can add GO annotations via step-by-step, quick add, or duplicate.
